test/tests/functional/pbs_mom_walltime.py

```diff
@@ -151,6 +151,8 @@ def test_suspend_time_not_counted_in_walltime(self):
 
         # Sleep for the job's entire walltime secs so we can catch any
         # walltime increment during job suspension time
+        self.logger.info("Sleeping for job's entire walltime secs. Walltime " +
+                         "increments during job suspension will be caught")
```

Comment:
now that reading it again, I would adjust the log message to "suspending the job for 30s, job's execution time. walltime should not get incremented while job is suspended"

Reply:
@anamikau , Changed log message as requested. Please find test logs attached